rotkehlchen/db/dbhandler.py

```diff
@@ -3610,3 +3610,23 @@ def rows_missing_prices_in_base_entries(
                     f'with asset identifier {asset_name}. {str(e)}',
                 )
         return result
+
+    def get_entries_assets_history_events(
```

Comment:
nitpick: Should move functions like this to own file. dbhanler.py is huge. This is why we have been breaking it up in other  files.